go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/uber/h3-go/v4
 
-go 1.20
+go 1.21
```

Comment:
Again, why change the minimum version here? Are you using a feature that only exists in 1.21?

Reply:
Not really, usually most of the libraries support 3 latest Go versions. Makes it easier for maintainers to maintain the package.
So it depends on h3-go’s backward comptability policies.
I can downgrade to Go 1.20 as well.

Reply:
Got it. As I said, I don't develop in Go, so I will leave that decision to @zachcoleman and/or @jogly

Reply:
I only care we support the versions of Go that Go cares to support (https://go.dev/doc/devel/release), and their policy is 2 versions. Go is at 1.23 now so I support dropping 1.20 😁